test/specs/products.js

```diff
@@ -197,5 +226,62 @@ describe('Products Page Tests', () => {
             // console.log('expected prices: ', dictValues)
         });
     })
-})
 
+    describe('Footer Tests', () => {
+
+        it("check Footer UI elements", async () => {
+            await LoginPage.open();
+            await LoginPage.login(CREDENTIALS.standard, CREDENTIALS.password);
+
+            expect(await ProductsPage.twitterIcon.isDisplayed()).toBe(true);
+            expect(await ProductsPage.facebookIcon.isDisplayed()).toBe(true);
+            expect(await ProductsPage.linkedInIcon.isDisplayed()).toBe(true);
+            expect(await ProductsPage.copyright.isDisplayed()).toBe(true);
+            expect(await ProductsPage.copyright.getText()).toContain("© 2022 Sauce Labs. All Rights Reserved. Terms of Service | Privacy Policy");
+            expect(await ProductsPage.robotFooter.isDisplayed()).toBe(true);
+            expect(await ProductsPage.robotFooter.getAttribute('src')).toEqual(ImageSource.footerRobot);
+
+            await ProductsPage.logout();
+        });
+
+        it("the 'Twitter' icon should redirect to the 'https://twitter.com/saucelabs' url", async () => {
+            await LoginPage.open();
+            await LoginPage.login(CREDENTIALS.standard, CREDENTIALS.password);
+            await ProductsPage.twitterIcon.waitForClickable()
+            await ProductsPage.twitterIcon.click()
+            await browser.switchWindow("https://twitter.com/saucelabs")
+
+            expect(await browser.getUrl()).toEqual("https://twitter.com/saucelabs");
+            await browser.closeWindow()
+            await browser.switchWindow('Swag Labs')
```

Comment:
You can put these in a method called closeWindowAndSwitchBack or something like that.